### PR TITLE
added login page along with styles

### PR DESCRIPTION
**Pull request by poojakelkar**
sign in
- Added Login page

### CODE REVIEW

File: src/pages/Login/styles.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    border: 1px solid white;
+    width: 30rem;
+    height: 20rem;
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    border-radius: 5px;
+}
+.SignIn-Title {
```
Comment (viru2001) [inline src/pages/Login/styles.css:39]: you should follow consistent naming convention while giving class names

**Review by viru2001 — APPROVED** [review]
everything looks good. 

left one comment

File: src/pages/Login/styles.css (hunk 1 above)
Comment (poojakelkar) [reply src/pages/Login/styles.css:39]: Ok, thanks for review

**Review by sakshi006 — APPROVED** [review]
Liked the naming convention.
Good to merge

**Pull request merged**